### PR TITLE
Build phishing classifier

### PR DESCRIPTION
**Pull request by erezh31**
## Status
In Progress - need to:
- [ ] add playbook & test playbook
- [ ] test with @Itay4 branch
- [ ] add description (with a technical writer)

## Description
A new version - training phishing classifier in Demisto

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](link)
- [ ] [CHANGELOG](link)

### CODE REVIEW

File: Scripts/DBotBuildPhishingClassifier/DBotBuildPhishingClassifier.yml [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  isArray: false
+  name: PhishingLabels
+  required: false
+  secret: false
+comment: Create a phishing classifier
+commonfields:
+  id: DBotBuildPhihsingClassifier
+  version: -1
+enabled: false
+name: DBotBuildPhihsingClassifier
```
Comment (Itay4) [inline Scripts/DBotBuildPhishingClassifier/DBotBuildPhishingClassifier.yml]:
```suggestion
name: DBotBuildPhishingClassifier
```

File: Scripts/DBotBuildPhishingClassifier/DBotBuildPhishingClassifier.yml [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    merge "inner communitcation" and "external legit email" to one tag called "non-malicious".
+    The input will be: malicious, credentials harvesting:malicious, inner communitcation:non-malicious,
+    external legit email:non-malicious'
+  isArray: false
+  name: PhishingLabels
+  required: false
+  secret: false
+comment: Create a phishing classifier
+commonfields:
+  id: DBotBuildPhihsingClassifier
```
Comment (Itay4) [inline Scripts/DBotBuildPhishingClassifier/DBotBuildPhishingClassifier.yml]:
```suggestion
  id: DBotBuildPhishingClassifier
```

**Comment by Itay4** [comment]
@kirbles19 please review the yamls

File: Playbooks/playbook-DBot_Create_Phishing_Classifier_V2.yml [hunk 3]
```diff
@@ -0,0 +1,384 @@
...
+    simple: emailclassification
+  required: false
+  description: Incident key expression to extract email tag
+- key: phishingLabels
+  value:
+    simple: '*'
+  required: false
+  description: 'Comma-separated values of email tags values and mapping. The script
+    going to consider only the tags specify in this field. You can map label to another
+    value by using this format: LABEL:MAPPED_LABEL. For example: let''s say we have
```
Comment (Itay4) [inline Playbooks/playbook-DBot_Create_Phishing_Classifier_V2.yml]: still have here two `'`

File: Playbooks/playbook-DBot_Create_Phishing_Classifier_V2.yml [hunk 4]
```diff
@@ -0,0 +1,384 @@
...
+    4 values in email tag: malicious, credentials harvesting, inner communitcation,
```
Comment (Itay4) [inline Playbooks/playbook-DBot_Create_Phishing_Classifier_V2.yml]: there are 5 values here

File: Scripts/DBotBuildPhishingClassifier/DBotBuildPhishingClassifier.py [hunk 5]
```diff
@@ -0,0 +1,33 @@
+from CommonServerPython import *
```
Comment (Itay4) [inline Scripts/DBotBuildPhishingClassifier/DBotBuildPhishingClassifier.py]: need to import here base64 in order to execute the script without commonserver, as done in the feature

**Comment by kirbles19** [comment]
@erezh31 Done.

**Pull request merged**